src/Grid.js

```diff
@@ -0,0 +1,133 @@
+import Cell from './Cell.js';
+
+export default class Grid
+{
+	constructor()
+	{
+		this.cycle = 0;
+		this.cells = [];
```

Comment:
@dominiktilp I have read about the # notation, but since this is aproposal yet, I did not want to use it. Instead I have read the principles behind it https://crockford.com/javascript/private.html